lua/terrortown/entities/items/item_ttt_speedrun.lua

```diff
@@ -9,11 +9,12 @@ ITEM.EquipMenuData = {
 }
 ITEM.CanBuy = {ROLE_TRAITOR, ROLE_DETECTIVE}
 
-ITEM.hud = Material("vgui/ttt/perks/hud_speedrun.png")
 ITEM.material = "vgui/ttt/icon_speedrun"
 
-hook.Add("TTTPlayerSpeedModifier", "SpeedrunModifySpeed", function(ply, _, _, refTbl)
-	if not IsValid(ply) or not ply:HasEquipmentItem("item_ttt_speedrun") then return end
+if SERVER then
+	hook.Add("TTTPlayerSpeedModifier", "TTT2SpeedRun", function(ply, _, _, speedMultiplierModifier)
+		if not IsValid(ply) or not ply:HasEquipmentItem("item_ttt_speedrun") then return end
 
-	refTbl[1] = refTbl[1] * 1.5 * (ply.speedrun_mul or 1)
```

Comment:
Why removed `speedrun_mul`?

Reply:
Because I don't see a reason to keep this. Why do we need this?

Reply:
https://github.com/TTT-2/ttt2h-pack-default/blob/a110bc1a5fe614945bda6c38ca798bf3b5be1257/lua/classes/classes/class_dazzle.lua#L16

Reply:
I know and I already updated the class locally since (at least in my opinion) it is cleaner to use the hook in the dazzle as well instead of a custom hacky variable.